features/create_filter.feature

```diff
@@ -414,3 +468,43 @@ Feature: Filters Private Endpoints Not Enabled
     """
 
     And the HTTP status code should be "400"
+
+  Scenario: Creating a filter based on Cantabular blob
+    When I POST "/filters"
+    """
+      {
+      "dataset":{
+          "id":      "cantabular-table-example",
+          "edition": "2021",
+          "version": 1
+      },
+      "population_type": "Example",
+      "dimensions": [
+        {
+          "name": "siblings",
+          "options": [
+            "0-3",
+            "4-7",
+            "7+"
+          ],
+          "is_area_type": false
+        },{
+          "name": "geography",
+          "options": [
+            "Cardiff",
+            "London",
+            "Swansea"
+          ],
+          "is_area_type": true
+        }
+      ]
+    }
+    """
+    Then I should receive the following JSON response:
```

Comment:
Prior to this check, I would also ensure that the response code is `400`

And the HTTP status code should be "400"
```suggestion
    Then the HTTP status code should be "400"
    And I should receive the following JSON response:
```